CMakeLists.txt

```diff
@@ -258,7 +258,7 @@ if(USE_OPTIM)
 	add_definitions(-DOPTIM_ENABLE_EIGEN_WRAPPERS)
 endif(USE_OPTIM)
 
-OPTION(USE_CPPAD "Use CppAD and CppADCodeGen" OFF)
+OPTION(USE_CPPAD "Use CppAD and CppADCodeGen" ON)
```

Comment:
Best not to enable it by default, let the user enable it, if needed (TinyDifferentiableSimulator should compile without all the dependencies, just using TinyAlgebra etc).

Reply:
Fixed in  2726923.